proposals/NNNN-atf_facade_proposal.md

```diff
@@ -0,0 +1,255 @@
+# ATF Facade
+
+* Proposal: [SDL-NNNN](NNNN-atf_facade_proposal.md)
+* Author: [Oleksandr Deriabin](https://github.com/aderiabin)
+* Status: **Awaiting review**
+* Impacted Platforms: [ATF]
+
+## Introduction
+
+This proposal describes unified interface for ATF (Automated Test Framework) - ATF Facade.
+ATF should provide unified and flexible interface for creation test scripts to perform checks of SDL functionality.
+
+Main ATF Facade purposes:
+ - Simplify access to ATF features for test scripts
+ - Separate test scripts from ATF
+ - Provide well-structured intuitive interface of ATF for test scripts
+ - Unify way of use ATF for creating automated tests
```

Comment:
Please update to "Unify way of using ATF for creating automated tests"

Reply:
Updated (see 9943914522e19530a121b117461502d1961c2b27)